enterprise/docker-compose.yml

```diff
@@ -2,14 +2,21 @@
 # Please take a look at the README at the top of this repo or the regular docs for more info.
 
 services:
+  # MongoDB: https://hub.docker.com/_/mongo/
   mongodb:
-    image: "mongo:5.0"
+    image: "mongo:6.0.18"
```

Comment:
We shouldn't use a specific patch version. Let's use `6.0` here. That way, people always get the latest 6.0 stable version.